### ISSUE TITLE
Propose the administration panel for isso to isso

### ISSUE THREAD

**Issue by Hywan** [hoaproject/ActionBoard#4]
Depends on #3.

Because we love open-source, we don't want to do something only for us, and because doing something public makes it better, we would like to propose the administration panel for isso to isso.
#### Tasks
- [ ] Create an issue to see if they are interested,
- [ ] If they are, discuss about the lead and the hosting,
- [ ] Finally, promote it.

**Comment by Hywan** [comment]
Still 2 days left :-). Tic tac, tic tac :clock11:.

**Comment by Hywan** [comment]
@Pierozi Any news? Would you like to postpone to the next milestone (in a couple of days)?

**Comment by Hywan** [comment]
ping?

**Comment by Hywan** [comment]
@Pierozi ping?
